pnet_macros/src/decorator.rs

```diff
@@ -596,9 +596,9 @@ fn handle_vector_field(cx: &mut GenContext,
                                 pub fn get_{name}_raw(&self) -> &[u8] {{
                                     let _self = self;
                                     let current_offset = {co};
-                                    let len = {packet_length};
+                                    let end = current_offset + {packet_length};
```

Comment:
When making changes to pnet_macros, I'd appreciate it if you could add test cases to the relevant directories in pnet_macros/tests, so we don't end up getting regressions.